pkg/wasm/httpfetcher.go

```diff
@@ -95,19 +95,36 @@ func (f *HTTPFetcher) Fetch(ctx context.Context, url string, allowInsecure bool)
 			continue
 		}
 		if resp.StatusCode == http.StatusOK {
-			body, err := io.ReadAll(resp.Body)
-			resp.Body.Close()
+			// Limit wasm module to 256mb; in reality it must be much smaller
+			body, err := io.ReadAll(io.LimitReader(resp.Body, 1024*1024*256))
```

Comment:
can you rebase? these are from the other PR which si now merged, shouldn't be here